.gitignore

```diff
@@ -6,4 +6,3 @@
 .idea
 npm-debug.log
 node_modules
-dist
```

Comment:
Sorry man, but you can't remove this. When the package is released, the `dist` will be published, but on GitHub doesn't need this folder